github/repos_contents.go

```diff
@@ -119,6 +120,10 @@ func (s *RepositoriesService) DownloadContents(owner, repo, filepath string, opt
 	return nil, fmt.Errorf("No file named %s found in %s", filename, dir)
 }
 
+func escapeURLPath(path string) string {
+	return template.URLQueryEscaper(path)
```

Comment:
> It's also interesting to note that the GitHub API itself does not escape the '#' in URLs it outputs (e.g: the response from https://api.github.com/repositories/12052373/contents/golang_tour). 

@willnorris This should now be fixed, but let us know if you're still noticing this anywhere. Thanks again for the report! 💙